packages/compass-e2e-tests/tests/collection-aggregations-tab.test.ts

```diff
@@ -75,7 +98,7 @@ describe('Collection aggregations tab', function () {
     ]);
   });
 
-  // the aggregation runs and the preview is shown
+  // TODO: we can probably remove this one now that there is a more advanced one. or merge that into here?
```

Comment:
I'm in two minds about keeping this simple case around or removing it given that the more complicated one does all the same things and more.